Tests/AccessibilityTests/accessibility-fss-ui-simplifiedsearch-page.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { chromium, BrowserContext, Browser, Page } from 'playwright'
+import { injectAxe,checkA11y } from 'axe-playwright'
+const { autoTestConfig } = require('../FunctionalTests/appSetting.json');
+const { pageObjectsConfig, pageTimeOut } = require('../FunctionalTests/pageObjects.json');
+import { LoginPortal} from '../FunctionalTests/helpermethod'
+
+let browser: Browser
+let context: BrowserContext;
+let page: Page
+
+describe('FSS UI Simplified Search Page Accessibility Test Scenarios', () => {
+  jest.setTimeout(pageTimeOut.timeOutInMilliSeconds);
+  beforeAll(async () => {
```

Comment:
Please split the `beforeAll`/`afterAll` methods

Reply:
Hi Sara, In this case we are not checking the functionality rather checking the accessibility compliance of the dynamic html of the page.